packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -123,18 +129,55 @@ const PopoverButton = ({ onClick, disabled, children }: PopoverButtonProps) => {
 };
 
 interface EntryValidationTextProps {
-  status: ReleaseAction['entry']['status'];
   action: ReleaseAction['type'];
+  schema: Schema.ContentType;
+  components: { [key: Schema.Component['uid']]: Schema.Component };
+  entry: ReleaseActionEntry;
 }
 
-const EntryValidationText = ({ status, action }: EntryValidationTextProps) => {
+const EntryValidationText = ({ action, schema, components, entry }: EntryValidationTextProps) => {
   const { formatMessage } = useIntl();
+  const { validate } = unstable_useDocument();
+
+  const getValidationErrors = () => {
+    try {
+      validate(entry, { contentType: schema, components });
+
+      return {};
+    } catch (error) {
+      return getYupInnerErrors(error as ValidationError);
+    }
+  };
+
+  const validationErrors = getValidationErrors();
```

Comment:
I'm not sure about this. Isn't this going to run on every render? Shouldn't we only run it on mount or when clicking the refresh button.

Reply:
You're right but <EntryValidationText> should not rerender in any scenario because it doesn't has any state logic inside, I make some tests with console.logs inside validate function and looks like it's running once for each entry and that's fine